### PR TITLE
Ribn 562

### PR DESCRIPTION
**Pull request by KalervoHyyppa**
Updated some widgets to be more responsive and added NFT asset card

### CODE REVIEW

**Comment by ultimaterex** [comment]
@KalervoHyyppa Please change target to RC-5 instead of Main

File: lib/widgets/molecules/asset_card.dart [hunk 1]
```diff
@@ -58,52 +59,59 @@ class AssetCard extends StatelessWidget {
...
-                            color: RibnColors.primary,
-                            fontWeight: FontWeight.w600,
-                            fontFamily: 'DM Sans',
-                            fontStyle: FontStyle.normal,
-                            fontSize: 10.4,
-                            decoration: TextDecoration.underline,
-                            letterSpacing: 1,
+              iconImage,
+
+              SizedBox(width: 20),
```
Comment (Brianspha) [inline lib/widgets/molecules/asset_card.dart:67]: We could add const SizedBox(width:20) else. The linter is going to throw a warning

**Review by Brianspha — CHANGES_REQUESTED** [review]
LGTM

**Comment by KalervoHyyppa** [comment]
@ultimaterex Ribn targets the main branch in pubspec (Ribn needs access to this change), should I make this PR to dev and then dev to main?

**Comment by ultimaterex** [comment]
> @ultimaterex Ribn targets the main branch in pubspec (Ribn needs access to this change), should I make this PR to dev and then dev to main?

Correct, pending changes go to dev and when we "release" it goes into main

**Review by ultimaterex — APPROVED** [review]
✅

**Review by Brianspha — APPROVED** [review]

**Pull request merged**